### DISCUSSION TITLE
How is Thruster powered by the EPS

### DISCUSSION THREAD

**Discussion by kodsurf** [RobertK66/obc_1769_core#65] [General]
![image](https://user-images.githubusercontent.com/66519643/206857069-717b732b-7baa-4f76-b83d-4efc934332df.png)
Pinnout of PSU unit

Where are original docu on PSU unit ? (this image is from PSU git repo)

Thruster needs 12V supply. 

EPS provides 3.3V and 5.0V 

How the thruster is powered ?

**Comment by RobertK66** [comment]
This is not a Software Issue. This is Hardware and System Design. So maybe someone (@WolfgangTreb , @AndyFHWN) could start a place to document this kind of discussions !? 

There should be an Hardware design somewhere - at least showing the Power supply rails on the bus sides X-,X+/Y-/Y+ ....
